src/components/dropdown/index.coffee

```diff
@@ -35,15 +35,10 @@ positionDropdown = ({selection, dropdown, useScroll}, {align, matchWidth}) ->
     scrollbarSize(),
   )
 
-  yPos = 'top'
   if not parentFixed
     x += window.scrollX || window.pageXOffset
     y += window.scrollY || window.pageYOffset
 
-    if direction is 'up'
-      yPos = 'bottom'
-      y = document.body.clientHeight - y - dropdownRect.height
```

Comment:
This code is designed to allow the margins to be applied to space the dropdown correctly in relation to the dropdown. 

Before this change (and changing the `--dropdown-spacing` to 15px):

*Dropdown down*: - Spacing shown between button and dropdown
![Screenshot 2019-12-04 at 15 02 51](https://user-images.githubusercontent.com/3194349/70153780-68186480-16a7-11ea-9e82-b84af3a9a3a7.png)

*Dropdown up*: - Same dropdown, not enough space to show downwards so flip it upwards
![Screenshot 2019-12-04 at 15 02 58](https://user-images.githubusercontent.com/3194349/70153778-68186480-16a7-11ea-8d38-ee6a620683d5.png)


After the change:
![Screenshot 2019-12-04 at 15 03 32](https://user-images.githubusercontent.com/3194349/70153922-9a29c680-16a7-11ea-9e8c-3e5462f32d17.png)

Essentially, as we apply `margin-bottom` it means the dropdown is now shown without spacing.

I'm sure there's a way to resolve this with changing the CSS, worth fiddling with it 😉